docs/source/data/subscriptions.mdx

```diff
@@ -0,0 +1,508 @@
+---
+title: Subscriptions
+description: Persistent GraphQL operations
+---
+
+import {ExpansionPanel} from 'gatsby-theme-apollo-docs';
+
+> **Subscriptions are not supported in [Apollo Federation](https://www.apollographql.com/docs/federation/).**
+
+**Subscriptions** are GraphQL operations that maintain an active connection between client and server. This enables your server to _push_ updates to a subscription's result as they occur, instead of requiring your client to _poll_ for updates.
+
+Because subscriptions use persistent two-way communication, they usually use [the WebSocket protocol](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API) instead of HTTP. To support this, Apollo Server lets you [set a subscription-specific endpoint](#setting-a-subscription-endpoint) that's separate from the default endpoint for queries and mutations.
+
+You can use subscriptions with the core `apollo-server` library, or with any of Apollo Server's supported [middleware integrations](#using-with-middleware-integrations).
+
+> **Important:** Compared to queries and mutations, subscriptions are significantly more complex to implement. Before you begin, [confirm that your use case requires subscriptions](https://www.apollographql.com/docs/react/data/subscriptions/#when-to-use-subscriptions).
+
+## Schema definition
+
+You define your schema's supported subscriptions as fields of the `Subscription` type:
```

Comment:
This may not be a distinction we care about, but in the spec the fields defined on `Subscription` wouldn't themselves be called subscriptions, similar to how fields on `Query` aren't queries. These are root fields that get used in subscription (respectively query) operations. I realize referring to these as root fields makes this overly technical, and because subscription operations can only [include a single root field](http://spec.graphql.org/draft/#sec-Single-root-field) there is a one to one correspondence. So not sure if we need to be struct about terminology here.

Reply:
Yeah totally, I'll work on this. Also I wasn't aware sub operations can only have one root field! Makes a lot of sense.